config/serverless.oblt.yml

```diff
@@ -7,6 +7,7 @@ xpack.infra.enabled: false
 xpack.securitySolution.enabled: false
 xpack.uptime.enabled: false
 xpack.legacy_uptime.enabled: false
+xpack.screenshotting.enabled: false
```

Comment:
Given you're already overriding the option in the global `config/serverless.yml` config file, why are you also doing in in (2 of the 3) project configuration files?

Reply:
OHO! :TIL: thank you!